Testing/SystemTests/tests/framework/AlgorithmMDWorkspaceValidateInputsTest.py

```diff
@@ -6,22 +6,23 @@
 # SPDX - License - Identifier: GPL - 3.0 +
 from mantid.simpleapi import CreateMDWorkspace
 
-from AlgorithmDialogsStartupTestBase import AlgorithmDialogsStartupTestBase
+from AlgorithmValidateInputsTestBase import AlgorithmValidateInputsTestBase, INPUT_WS_NAME
 
 
-class AlgorithmDialogsMDWorkspaceStartupTest(AlgorithmDialogsStartupTestBase):
+class AlgorithmMDWorkspaceValidateInputsTest(AlgorithmValidateInputsTestBase):
     """
-    A system test for testing that the Algorithm Dialogs open ok with a MDWorkspace in the ADS.
+    A system test for testing that the Algorithm validateInputs method works with a MDWorkspace in the ADS.
     """
 
     def _setup_test(self) -> None:
         self._workspace_type = "MDWorkspace"
+        self._exclude_algorithms = ["CalculateMonteCarloAbsorption", "DNSMergeRuns", "HB3AIntegratePeaks", "Mean"]
```

Comment:
Why are we excluding these algorithms? I notice we do this for several of the tests.

Reply:
These algorithms are being excluded temporarily because they fail the `validateInputs` check. The plan is to fix these bugs and remove these exclude algorithms lists in a separate PR. I have opened an issue for this: https://github.com/mantidproject/mantid/issues/35686

They are currently failing because they each have one or more workspace properties such as:
```
self.declareProperty(
            MatrixWorkspaceProperty("SampleWorkspace", "", direction=Direction.Input), doc="Name of the Sample input Workspace"
        )
```
but they do not have a validator to ensure the correct workspace type is being passed to the property:
```
self.declareProperty(
            MatrixWorkspaceProperty("SampleWorkspace", "", direction=Direction.Input), doc="Name of the Sample input Workspace", validator=MatrixWorkspaceValidator()
        )
```